### ISSUE TITLE
Care about SEO ?  Use HTTPS

### ISSUE THREAD

**Issue by telmosilva** [chtefi/how-to-make-a-good-website#1]
Care about SEO ?

Use HTTPS

Why is that?

**Comment by sderosiaux** [comment]
HTTPS is a ranking signal since a few years.
For instance, see https://webmasters.googleblog.com/2014/08/https-as-ranking-signal.html

**Comment by telmosilva** [comment]
Yeah I knew that but last I heard its just if the URL has https in the string, not if its certificate is valid or even if it exists.  Has that changed?  http://searchengineland.com/googles-https-algorithm-two-years-later-still-looks-url-give-ranking-boost-261156

**Comment by sderosiaux** [comment]
Honestly, I have no idea if they changed this behavior, who knows. ;)

Their HTTPS-everywhere politics is quite clear anyway: certificates with SHA-1 algorithm are blocked (2016), HTTP pages with passwords are now marked unsecured (2017). I guess their SEO algorithms should follow the same rules.
